### ISSUE TITLE
Error validation on all forms to be changed to WET validation instead of Elgg

### ISSUE THREAD

**Issue by piet0024** [gctools-outilsgc/gcconnex#941]
When an error occurs with the form validation (ex when creating a blog and not filling out required fields) it should interrupt the user and tell them how to fix the error. Currently a page load is required, the error checking happens and appears in a system message that does not get read by screen readers.

The focus on the screen reader then goes to the bottom input and does not allow the user to correct any errors. 

Validation should happen before the page loads

@CoolColin

**Comment by piet0024** [comment]
Event calendar validation should be looked at for the date. User can free type characters in the date  and if the date is incorrect it simply says "fix the required fields". Example: end date can be set to the past.
